src/DisplayBoard.hpp

```diff
@@ -0,0 +1,9 @@
+#pragma once
+#include <sstream>
+#include <iostream>
+#include "Board.hpp"
+
+namespace DisplayBoard
+{
+    std::string displayInterface(const Board& b);
```

Comment:
This wouldn't be necessary if `drawBoard` had been publicly visible.

Reply:
rewritten by @jzych